src/main/resources/avro/genotypephenotypemethods.avdl

```diff
@@ -0,0 +1,319 @@
+
+@namespace("org.ga4gh.methods")
+/**
+This protocol defines genotype to phenotype association
+query system.
+
+Suggested ontologies used in type-based queries include:
+
+Phenotype:
+Human Phenotype Ontology (HPO): http://www.ontobee.org/browser/index.php?o=hp
+Disease Ontology (DO): http://purl.obolibrary.org/obo/DOID_4
+
+Feature:
+Sequence Ontology (SO): http://www.sequenceontology.org/browser/
+
+Evidence:
+Evidence Code Ontology (ECO): http://www.ontobee.org/browser/index.php?o=ECO
+Ontology for Biomedical Investigations (OBI): http://purl.obofoundry.org/obo/obi/browse
+*/
+protocol GenotypePhenotypeMethods {
+
+
+import idl "genotypephenotype.avdl";
+import idl "methods.avdl";
+import idl "common.avdl";
+
+/******************  /phenotypeassociationsets/search  *********************/
+/** This request maps to the body of `POST /phenotypeassociationsets/search` as JSON. */
+record SearchPhenotypeAssociationSetsRequest {
+  /**
+  The `Dataset` to search.
+  */
+  string datasetId;
+
+  /**
+  Specifies the maximum number of results to return in a single page.
+  If unspecified, a system default will be used.
+  */
+  union { null, int } pageSize = null;
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  To get the next page of results, set this parameter to the value of
+  `nextPageToken` from the previous response.
+  */
+  union { null, string } pageToken = null;
+}
+
+/** This is the response from `POST /phenotypeassociationsets/search` expressed as JSON. */
+record SearchPhenotypeAssociationSetsResponse {
+
+  /** The list of matching phenotype association sets. */
+  array<org.ga4gh.models.PhenotypeAssociationSet> phenotypeAssociationSets = [];
+
+  /**
+  The continuation token, which is used to page through large result sets.
+  Provide this value in a subsequent request to return the next page of
+  results. This field will be empty if there aren't any additional results.
+  */
+  union { null, string } nextPageToken = null;
+}
+
+/**
+Gets a list of `PhenotypeAssociationSet`s matching the search criteria.
+
+`POST /phenotypeassociationsets/search` must accept a JSON version of
+`SearchPhenotypeAssociationSetsRequest` as the post body and will return a JSON version
+of `SearchPhenotypeAssociationSetsResponse`.
+*/
+SearchPhenotypeAssociationSetsResponse searchPhenotypeAssociationSets(
+  /** This request maps to the body of `POST /phenotypeassociationsets/search` as JSON. */
+  SearchPhenotypeAssociationSetsRequest request) throws GAException;
+
+
+/******************  /phenotypes/search  *********************/
+
+/**
+One or more phenotypes can be queried together.
+*/
```

Comment:
Since there's no `PhenotypeAssociationSet` mentioned in this or other queries, I assume they can span across multiple such sets?